app/src/ui/check-runs/ci-check-run-list.tsx

```diff
@@ -104,24 +101,88 @@ export class CICheckRunList extends React.PureComponent<
     this.props.onCheckRunClick?.(checkRun)
   }
 
-  private renderList = (): JSX.Element | null => {
-    const list = [...this.props.checkRuns]
-      .sort((a, b) =>
-        getCheckRunDisplayName(
-          a,
-          this.state.checkRunsHaveMultipleEventTypes
-        ).localeCompare(
-          getCheckRunDisplayName(b, this.state.checkRunsHaveMultipleEventTypes)
-        )
-      )
+  private getCheckRunsGroupedByWorkflowAction = (): _.Dictionary<
+    IRefCheck[]
+  > => {
```

Comment:
Is `_.groupBy` the reason for not using a `Map<string, IRefCheck[]>` instead? AFAICT we don't use `_.Dictionary` anywhere else 🤔 

This is not an issue, it just called my attention 😅

Reply:
Well in the next PR.. I get rid of it since the group logic got more complicated. And Yes, that lodash returns the lodash type of dictionary.. which is basically a map.